src/element.js

```diff
@@ -36,7 +36,21 @@ function findElement(sendSessionCommand, strategy, selector) {
             getText: getText.bind(null, sendElementCommand),
             getCss: getCss.bind(null, sendElementCommand)
         };
-    });
+    }).catch(() => undefined);
+}
+
+async function findElement(sendSessionCommand, strategy, selector, maxWaitTime = 0) {
```

Comment:
Yes but the goal of this package is to provide straight forward bindings for the spec. Not to create a framework. This package only provide a good base for a framework if needed. So currently I propose to move the utility function to your code or other open source project